packages/noco-docs/docs/070.fields/040.field-types/060.formula/070.formula-result-formatting.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Formatting Formula Results'
+description: 'This article explains how to format the output of formulas in NocoDB, covering various formatting options based on the result type.'
+tags: ['Fields', 'Field types', 'Formula', 'Formatting']
+keywords: ['Fields', 'Field types', 'Formula', 'Formatting', 'Create formula field', 'Formatting options']
+---
+
+Formatting the output of formulas allows you to tailor how your data is displayed based on the type of result. Various formatting options are available to ensure that your data is presented in the most useful and visually appealing way.
+
+![image](/img/v2/fields/types/formula-format.png)
+
+
+## Numeric Formats
+
+<img src="/img/v2/fields/numeric-format.png" alt="image" width="400"/>
+
+### Decimal
+The `Decimal format` is used to display results with a specified number of decimal places, allowing you to define the precision.
+
+### Currency
+The `Currency format` is ideal for displaying monetary values, with the option to set and display a specific currency symbol.
+
+### Percent
+The `Percent format` displays results as a percentage with the % symbol.
+
+### Rating
+The `Rating format` is ideal for ratings on a scale from 1 to 10. It allows you to customize the icon, color, and choose the maximum rating value within that range.
+
+
+
+## Date Formats
+
+<img src="/img/v2/fields/date-format.png" alt="image" width="400"/>
+
+### Date Time
+The `Date Time format` displays both date and time values.
+
+### Date
+The `Date format` displays only the date values, with a wide range of date formats available.
+
+### Time
+The `Time format` displays the result in either 12-hour or 24-hour time format.
+
+
+## Text Formats
+
+<img src="/img/v2/fields/text-format.png" alt="image" width="400"/>
+
+### Email
+The `Email format` is used to display results as clickable email links. When an email address is detected in the formula output, it will be formatted as a mailto link, allowing users to click and open their default email client to send an email.
+
+### URL
+The `URL format` is designed to display results as clickable web links. When a valid URL is recognized in the formula output, it will be formatted as a hyperlink, enabling users to click and navigate directly to the specified web address.
+
+### Phone Number
+The `Phone number format` is ideal for displaying the results as phone numbers. 
+
+
+## Boolean Format
+
+<img src="/img/v2/fields/boolean-format.png" alt="image" width="400"/>
+
+### Checkbox
+When a formula's output is recognized by NocoDB as a Boolean result, you can apply the `Checkbox format` to display the result similarly to a checkbox field. It allows you to customize the icon and color. Keep in mind that not all formulas will support this type of formatting. 
```

Comment:
see if this sounds better? last line can be removed IMO.

When NocoDB recognizes a formula's output as a Boolean result, you can apply the `Checkbox format` to display it in a manner similar to a checkbox field. Customize icon and color as required.